manim/mobject/types/vectorized_mobject.py

```diff
@@ -1060,15 +1067,25 @@ def get_points_defining_boundary(self):
         # Probably returns all anchors, but this is weird regarding  the name of the method.
         return np.array(list(it.chain(*[sm.get_anchors() for sm in self.get_family()])))
 
-    def get_arc_length(self, n_sample_points=None):
-        if n_sample_points is None:
-            n_sample_points = 4 * self.get_num_curves() + 1
-        points = np.array(
-            [self.point_from_proportion(a) for a in np.linspace(0, 1, n_sample_points)]
+    def get_arc_length(self, sample_points_per_curve: Optional[float] = None) -> float:
```

Comment:
`np.linspace` actually can't accept a float as the third argument.
```suggestion
    def get_arc_length(self, sample_points_per_curve: Optional[int] = None) -> float:
```